align/schema/types.py

```diff
@@ -242,3 +242,33 @@ def __setitem__(self, item, value):
 
     def __eq__(self, other):
         return self.__root__ == other
+
+    def __contains__(self, item):
+        return item in self.__root__
+
+    def update(self, other):
+        self.__root__.update(other)
+
+class String(str):
```

Comment:
@parijatm See Issue #795 for an alternative way to do this by inheriting from collections.abc.Sequence instead of str or collections.UserString